server/chat-jsx.tsx

```diff
@@ -36,3 +37,35 @@ export type VNode = preact.VNode;
 
 export const h = preact.h;
 export const Fragment = preact.Fragment;
+
+function subRender(
+	child: preact.ComponentChild, opts?: {isTrusted?: boolean, replaceLinebreaks?: boolean}
+): preact.VNode {
+	switch (typeof child) {
+	case 'string':
+		return <div dangerouslySetInnerHTML={{__html: Chat.formatText(child, opts?.isTrusted, opts?.replaceLinebreaks)}} />;
```

Comment:
A div is often the wrong choice for this...

Reply:
I wonder if a fragment with `dangerouslySetInnerHTML` is possible.

Reply:
Some of the examples I found online used divs so I went with those, I’ll switch to span if thats preferred though

Reply:
I think ideally users should just directly use `dangerouslySetInnerHTML` so you don't have an extra span _or_ an extra div.

Reply:
But if you really want to use this architecture, `span` would let chat messages be inline. Which they normally are, anyway.